tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -37,147 +37,279 @@
-        required=False
-    )
+    def add_profiling_related_args(parser):
+        profiling_related_args = parser.add_argument_group('Profiling related arguments')
+        profiling_related_args.add_argument(
+            '--profile',
+            help='activate metric profiling mode',
+            type=cast_to_bool,
+            required=False
+        )
+        profiling_related_args.add_argument(
+            '--profiler_logs_dir',
+            help='path to save profiler logs',
+            type=partial(get_path, is_directory=True),
+            default=Path.cwd(),
+            required=False
+        )
+        profiling_related_args.add_argument(
+            '--profile_report_type',
+            help='report type for profiler logs',
+            default='csv',
+            choices=['csv', 'json'],
+            required=False
+        )
 
-    parser.add_argument(
-        '--ignore_result_formatting',
-        help='allow to get raw metrics results without data formatting',
-        required=False,
-        default=False,
-        type=cast_to_bool
-    )
+    def add_tool_customization_args(parser):
+        tool_customization_args = parser.add_argument_group('Tool customization arguments')
+        tool_customization_args.add_argument(
+            '--progress',
+            help='progress reporter. You can select bar or print',
+            default='bar',
+            required=False
+        )
+        tool_customization_args.add_argument(
+            '--progress_interval',
+            help='interval for update progress if selected *print* progress.',
+            type=int,
+            default=1000,
+            required=False
+        )
+        tool_customization_args.add_argument(
+            '--ignore_result_formatting',
+            help='allow to get raw metrics results without data formatting',
+            type=cast_to_bool,
+            default=False,
+            required=False
+        )
+        tool_customization_args.add_argument(
+            '--stored_predictions',
+            help='path to file with saved predictions. Used for development',
+            # since at the first time file does not exist and then created we can not always check existence
+            required=False
+        )
+        tool_customization_args.add_argument(
+            '--csv_result',
+            help='file for results writing',
+            required=False,
+        )
+        tool_customization_args.add_argument(
+            '--intermediate_metrics_results',
+            help='enables intermediate metrics results printing',
+            type=cast_to_bool,
+            default=False,
+            required=False
+        )
+        tool_customization_args.add_argument(
+            '--metrics_interval',
+            help='number of iteration for updated metrics result printing',
+            type=int,
+            default=1000,
+            required=False
+        )
+        tool_customization_args.add_argument(
```

Comment:
only ov support async mode now, so make sense to move OV section async related args

Reply:
Moved